src/ray/raylet/agent_manager.cc

```diff
@@ -73,11 +50,6 @@ void AgentManager::StartAgent() {
   argv.push_back("--agent-id");
   argv.push_back(agent_id_str.c_str());
 
-  // Disable metrics report if needed.
-  if (!RayConfig::instance().enable_metrics_collection()) {
-    argv.push_back("--disable-metrics-collection");
```

Comment:
We already deleted this param?

Reply:
I moved this into node_manager.cc since runtime env agent doesn't need this parameter